### PR TITLE
feat: Notification UI

### PR DESCRIPTION
**Pull request by borichellow**
[AR-1165 ](https://wearezeta.atlassian.net/browse/AR-1165)

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [ ] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [ ] contains a reference JIRA issue number like `SQPIT-764`
  - [ ] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Added UI part for notifications. 
So when it will work, the app can display it correctly. 

### Attachments (Optional)

photo5334596931118282769

photo5334596931118282768

photo5334596931118282767

photo5334596931118282766

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: app/src/main/kotlin/com/wire/android/notification/NotificationManager.kt [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+            setWhen(conversation.lastMessageTime)
+
+            setLargeIcon(conversation.image?.toBitmap())
+
+            addAction(getActionCall(conversation))
+            addAction(getActionReply(conversation.id))
+
+            val body = getFormattedText(conversation.messages, !conversation.isDirectChat)
+            setContentText(body)
+            setStyle(NotificationCompat.BigTextStyle().bigText(body))
```
Comment (ohassine) [inline app/src/main/kotlin/com/wire/android/notification/NotificationManager.kt:142]: I think we need to use [MessagingStyle](https://developer.android.com/training/notify-user/expanded#message-style) here. It's created for messaging apps

**Review by ohassine — CHANGES_REQUESTED** [review]

File: app/src/main/kotlin/com/wire/android/notification/NotificationManager.kt (hunk 1 above)
Comment (borichellow) [reply app/src/main/kotlin/com/wire/android/notification/NotificationManager.kt:142]: i was thinking about it. But i didn't find a way to hide user avatar there. 
In MessagingStyle by default, the message has UserIcon, UserName and MessageBody, but in our [design](https://www.figma.com/file/UgKehvwIbiIiQ2CCEdsVuf/Android?node-id=5272%3A92311) there is no UserImage at the left side (only `LargeIcon` on the right side).

Comment (gongracr) [reply app/src/main/kotlin/com/wire/android/notification/NotificationManager.kt:142]: I think adapting the designs for the use of this `MessagingStyle` wouldn't be a problem

Comment (borichellow) [reply app/src/main/kotlin/com/wire/android/notification/NotificationManager.kt]: I talked to Wolfgang and he approved changed design :)

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -22,22 +22,25 @@
...
                 <action android:name="android.intent.action.VIEW" />
+
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
-                <data android:scheme="wire"
-                        android:host="access"/>
+
+                <data
+                        android:host="access"
+                        android:scheme="wire" />
```
Comment (gongracr) [inline app/src/main/AndroidManifest.xml:43]: Is this properly formatted?

Comment (borichellow) [reply app/src/main/AndroidManifest.xml:43]: that's how my AS formatted it :) reverted

**Review by ohassine — APPROVED** [review]
lgtm 🤘

**Comment by gongracr** [comment]
Code wise looks good to me! However due to some known Gradle issues I could not build and test it on my device yet. I'll try to do it tomorrow morning

**Pull request merged**